sources/packages/backend/apps/api/src/route-controllers/report/report.aest.controller.ts

```diff
@@ -81,7 +79,11 @@ export class ReportAESTController extends BaseController {
       const reportData = await this.reportService.getReportDataAsCSV(
         submissionResult.data.data,
       );
-      this.streamFile(response, payload.reportName, reportData);
+      this.reportControllerService.streamFile(
```

Comment:
Is it correct to assume that in the next PR this entire method will be moved to the controller service?

Reply:
Yes, I believe so.